CHANGELOG.md

```diff
@@ -15,6 +15,10 @@
 * Linting got around 20% faster due to the praisworthy performance
   improvements done in the [SwiftSyntax](https://github.com/apple/swift-syntax)
   library.
+* Add `no_empty_block` default rule to validate that code blocks should not be empty.
+  They should at least contain a comment.
```

Comment:
```suggestion
  They should at least contain a comment.  
```

Please append two spaces to the end of the description. They are required to enforce a line break in the rendered Markdown.